src/home.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Navigation } from 'react-native-navigation';
+import { getUserToken } from './components/cache';
+
+const Home = () => {
+
+  const [token, setToken] = useState('');
+  useEffect(() => {
+    const getInfo = async () => {
```

Comment:
talvez algo mais descritivo? como 
```suggestion
    const loadUserToken = async () => {
```
ou 
```suggestion
    const getToken = async () => {
```
?

Reply:
O `loadUserToken` eu achei uma boa